blackjax/inference/metrics.py

```diff
@@ -63,23 +63,15 @@ def gaussian_euclidean(
     ndim = jnp.ndim(inverse_mass_matrix)
     shape = jnp.shape(inverse_mass_matrix)[:1]
 
-    if ndim == 1:  # diagonal mass matrix
+    if ndim > 2:
```

Comment:
Nice, I like this! You should raise an error if `ndim=0` as well. The following flow would work:

```python
if ndim == 1:
    dot, matmul = jnp.multiply, jnp.multiply
elif ndim==2:
    dot, matmul = jnp.dot, jnp.matmul
else:
    raise ValueError
    
# Everything else
```

Reply:
Will do